src/Refinement/RefinedTriangulations.jl

```diff
@@ -0,0 +1,126 @@
+
+
+struct RefinedTriangulation{Dc,Dp,A<:Triangulation{Dc,Dp},B<:RefinedDiscreteModel} <: Triangulation{Dc,Dp}
+  trian::A
+  model::B
+end
+
+function get_refined_model(t::RefinedTriangulation)
+  return t.model
+end
+
+# Wrap Triangulation API
+function Geometry.get_background_model(t::RefinedTriangulation)
+  get_background_model(t.trian)
```

Comment:
is `get_background_model(t.trian)` always equivalent (i.e., `===`) to `t.model.model` ?

Reply:
That is a good question... I think it should always be the case, right? Even considering the discussion we had about the types of `model.model` and `model.parent`, what is clear is that `model.model`should never be a RefinedModel. Is that correct? In that case I think what your asking should be always true.

Reply:
With the changes in [ecd87ba](https://github.com/gridap/Gridap.jl/pull/838/commits/ecd87bab1a1ecdb851e07349d6ad28c3d93d967c), we should be enforcing that this is always true.